pkg/datapath/linux/config/config.go

```diff
@@ -23,6 +23,7 @@ import (
 	"net"
 	"reflect"
 	"sort"
+	"strconv"
```

Comment:
Random note, but it seems like everything else in this file is using `fmt.Sprintf()`. Is there a specific motivation for either `strconv` or `fmt`?

Reply:
No particular reason. I will change to `fmt.Sprintf` for the sake of consistency.